packages/fluent/scss/toolbar/_layout.scss

```diff
@@ -0,0 +1,232 @@
+@use "../core/" as *;
+@use "_variables.scss" as *;
+@use "../icon/_variables.scss" as *;
+@use "../button/_variables.scss" as *;
+
+@mixin kendo-toolbar--layout() {
+
+    // Base
+    .k-toolbar {
+        margin: 0;
+        padding-inline: var( --kendo-toolbar-padding-x, #{$kendo-toolbar-padding-x} );
+        padding-block: var( --kendo-toolbar-padding-y, #{$kendo-toolbar-padding-y} );
+        border-width: var( --kendo-toolbar-border-width, #{$kendo-toolbar-border-width} );
+        border-style: solid;
+        box-sizing: border-box;
+        outline: 0;
+        font-family: var( --kendo-toolbar-font-family, #{$kendo-toolbar-font-family} );
+        font-size: var( --kendo-toolbar-font-size, #{$kendo-toolbar-font-size} );
+        line-height: var( --kendo-toolbar-line-height, #{$kendo-toolbar-line-height} );
+        list-style: none;
+        display: flex;
+        flex-flow: row wrap;
+        gap: var( --kendo-toolbar-spacing, #{$kendo-toolbar-spacing} );
+        align-items: center;
+        justify-content: flex-start;
+        position: relative;
+        overflow: hidden;
+        -webkit-touch-callout: none;
+        -webkit-tap-highlight-color: $rgba-transparent;
+
+        // Remove once we decide to not size empty containers
+        &::before {
+            content: "";
+            height: $kendo-button-inner-calc-size;
+            margin-inline-start: -#{$kendo-toolbar-spacing};
+        }
+
+        &.k-toolbar-resizable {
+            flex-wrap: nowrap;
+        }
+
+        > * {
+            flex-shrink: 0;
+            display: inline-flex;
+            align-content: center;
+            vertical-align: middle;
+
+            > label {
+                margin-inline-end: var( --kendo-toolbar-spacing, #{$kendo-toolbar-spacing} );
+                align-self: center;
+            }
+        }
+
+        // Input
+        .k-input,
+        .k-picker {
+            width: var( --kendo-toolbar-input-width, #{$kendo-toolbar-input-width} );
+        }
+        .k-color-picker {
+            width: min-content;
+        }
+
+        // Overflow anchor
+        .k-overflow-anchor {
+            @include border-radius( 0 );
+            margin: 0;
+            padding-block: var( --kendo-toolbar-padding-y, #{$kendo-toolbar-padding-y} );
+            width: auto;
+            height: 100%;
+            aspect-ratio: 1;
+            border-width: 0;
+            border-color: inherit;
+            box-sizing: border-box;
+            justify-content: center;
+            position: absolute;
+            top: 0;
+            right: 0;
+
```

Comment:
you can use inset-block-start and inset-inline-end instead of top and right. In this way you'll be able to remove the RTL styles for the overflow-anchor